### ISSUE TITLE
Main Screen Plugins not Working

### ISSUE THREAD

**Issue by mechPenSketch** [godotengine/godot-docs#3672]
**Your Godot version:**
3.2

**Issue description:**
When I followed the instructions and activated my plugin, the main screen plugins (with 2D, 3D, Script and Asset Lib) is not updated to include the new tab.

**URL to the documentation page:**
https://github.com/godotengine/godot-docs/blob/master/tutorials/plugins/editor/making_main_screen_plugins.rst

**Comment by mechPenSketch** [comment]
I thought this was an issue because when I opened 2.5D demo project in the example provided the first time, I got an error message. However, when I opened it the second time, no error message appeared and the main screen plugin is updated with one more tab for 2.5D.

Now, I'm currently debugging my script. I suspect the problem lies in this line:
```
get_editor_interface().get_editor_viewport().add_child(main_panel_instance)
```
For now, _enter_tree() is working as ```print(0)``` shows a output upon activating the plugin. Printing ```main_panel_instance``` also return an object, which is the scene root of the main panel.
